### ISSUE TITLE
Deployment improvements

### ISSUE THREAD

**Issue by dchlab** [dchlab/ptt#8]
The conveniences and disadvantages of building/deploying the small app through a setup.exe like Inno Setup (for now, we aren't even talking about building the application with PyInstaller) :

Conveniences | Disadvantages
------------ | -------------
1 file package easy install | Windows Smart Screen (1st fence)
Easy uninstall | Antivirus might think it's not safe (2nd fence)
No loss of data files when uninstalled (especially my_tasks.json) | Need to build the setup with "PrivilegesRequired=lowest" in the .iss (else, we'll need to run the setup with admin privileges...) (3rd fence)
Python files are frozen | Application is not signed/verified (Bonus)
Not depending of installing manually Python, PyQT... | Better to "upload" the setup to Virus Total so the setup.exe is known a bit by AV engines (and limit false suspicion)

TLDR, Windows and antiviruses like Trend (not using it but used at my workplace) doesn't trust much my application which i'd prefer to deliver it as "packaged a little".

The best of the best would be to just drag and drop the files and run the scripts... but the setup.exe/package form is easier for the user when it's about to install/uninstall without making a mess. But PyQT5 dependancies also not helping... and i'd prefer having my own folders (ui, data etc...)

Proposal : either to sign the application and/or making Inno Setup .exe's more appealing, even not under admin privileges + maybe signing the setup.exe ? Or switching to .msi installers ?

**Comment by dchlab** [comment]
Fixed in release **0.3.2**.
The PTT setup (ptt-setup-version.exe) and the program itself (ptt.exe) executables are now signed with a PKCS12 (.pfx) certificate.
Since the application is dependant of PyQT libraries which needs to be installed, i think it's better to release the application this way.
NB : antivirus like Trend is now less picky when installing the application but still displays a small warning when installing an unknown/recent application, it's better than nothing i guess...
